ddtrace/__init__.py

```diff
@@ -5,6 +5,12 @@
 
 LOADED_MODULES = frozenset(sys.modules.keys())
 
+# Configuration for the whole tracer from file. Do it before anything else happens.
+from ddtrace.internal.native import get_configuration_from_disk
+
+for key, value in get_configuration_from_disk().items():
```

Comment:
Can we add an "internal" environment variable to gate-keep this feature. Something like `_DD_APM_STABLE_CONFIG_ENABLED`. This way we can make this feature opt-in until it's fully tested and validated. It will also allow us to introduce breaking changes and iterate on the initial design.

Reply:
I'm not sure this is necessary here; given that this change is a no-op if the stable config YAML files don't exist.